plugins/unmanaged_files/unmanaged_files_renderer.rb

```diff
@@ -35,7 +35,7 @@ def content(description)
               puts "Files: #{p.files}" if p.files
             end
           else
-            item "#{p.name} (#{p.type})"
+            item "#{p.name} (#{p.type})\n"
```

Comment:
That doesn't look correct. The `item` call should take care of adding the newline (or not) consistently across all scopes, that's the whole point of having the rendering framework around.